### DISCUSSION TITLE
Move the settings back to the menu bar

### DISCUSSION THREAD

**Discussion by albakhtari** [cstayyab/WALC#168] [Feature Requests]
Hello everyone,
I think that it's better if the devs move the settings location back to the menu bar, so that if there is an WhatsApp Web update it would not effect the main functionality of the WALC application.
For example, if you are a beta tester, then you cannot access the settings button because it's embedded in the webpage, therefore all the features are unusable and you have to go to the _settings.json_ file and edit it to suit your needs.
Right now the only feature that doesn't work is the message count, so if we get the menu bar back, then we can still use the other WALC features.
Thanks

**Comment by daffamumtaz2361** [comment]
I totally get that, it sucks that you can't change the settings when it seems only the read count is not working. But the reason it's changed to its own page is to allow more advanced kind of settings that would be difficult or impossible otherwise using a menu.

The absence of dashboard button is caused by broken integration with Whatsapp web, and it's only used to check if Whatsapp is loaded to put down the button. It is fixable and technically fixed in master. So when it releases, the button should always appear.

Also, you said you're beta tester? Please note that Whatsapp beta is not supported. A lot of features straight up not working. Not much I can do about that

**Reply by albakhtari** [reply]
I think you misunderstood me.
I meant to say, keep the settings window, but make the point of access, for example, File>Settings

**Comment by daffamumtaz2361** [comment]
Should be fixed in the new beta

**Reply by albakhtari** [reply]
Yep, all works!!
Thanks.
